1-Vanilla/main.js

```diff
@@ -1,231 +1,245 @@
-const list = [
+(function () {
+  // const list = []
+  const list = [
     {
       text: "Doing the Dishes",
       status: "unfinished",
-      isEditMode: false,
       id: 1,
     },
     {
       text: "Fold Laundry",
       status: "unfinished",
-      isEditMode: false,
       id: 2,
     },
     {
       text: "Build Todo App",
       status: "finished",
-      isEditMode: false,
       id: 3,
     },
     {
       text: "Get a haircut",
       status: "unfinished",
-      isEditMode: false,
       id: 4,
     },
     {
       text: "Call to mom and dad",
       status: "unfinished",
-      isEditMode: false,
       id: 5,
     },
     {
       text: "Game day against Elitzur Tel-Aviv",
       status: "unfinished",
-      isEditMode: false,
       id: 6,
     },
     {
       text: "Make dinner",
       status: "unfinished",
-      isEditMode: false,
       id: 7,
     },
     {
       text: "Eat lunch",
       status: "finished",
-      isEditMode: false,
       id: 8,
     },
     {
       text: "Go to Wix offices",
       status: "unfinished",
-      isEditMode: false,
       id: 9,
     },
     {
       text: "Watch Deni Avdjia game",
       status: "unfinished",
-      isEditMode: false,
       id: 10,
     },
     {
       text: "Go to step 2",
       status: "unfinished",
-      isEditMode: false,
       id: 11,
     },
     {
       text: "Infra webinar at 3",
       status: "unfinished",
-      isEditMode: false,
       id: 12,
     },
     {
       text: "Pay the bills",
       status: "unfinished",
-      isEditMode: false,
       id: 13,
     },
     {
       text: "Contribute code to open source",
       status: "unfinished",
-      isEditMode: false,
       id: 14,
     },
     {
       text: "Finish Velo site",
       status: "unfinished",
-      isEditMode: false,
       id: 15,
     },
   ];
-  
+
+  const editModeItems = [];
+
+  const ul = document.getElementById("todoList");
+
   const loadToDoList = (list) => {
-    const ul = document.getElementById("todoList");
     ul.innerHTML = "";
     list.forEach((item) => {
       ul.appendChild(createItem(list, item));
     });
   };
-  
+
   const addTodo = () => {
-    const inputText = document.getElementById("addTodo").value;
-    list.push({
-      text: inputText,
+    const inputText = document.getElementById("addTodo");
```

Comment:
consider assigning the `addTodo` input element to a variable in the root scope as well

Reply:
Alright.